build/download_k8s_proxy.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import packageInfo from "../package.json";
+import fs from "fs";
+import request from "request";
+import { ensureDir, pathExists } from "fs-extra";
+import path from "path";
+import { noop } from "lodash";
+import { isLinux, isMac } from "../src/common/vars";
+
+class K8sProxyDownloader {
+  public version: string;
+  protected url: string;
+  protected path: string;
+  protected dirname: string;
+
+  constructor(version: string, platform: string, arch: string, target: string) {
+    this.version = version;
+    this.url = `https://github.com/lensapp/lens-k8s-proxy/releases/download/v${this.version}/lens-k8s-proxy-${platform}-${arch}`;
+    this.dirname = path.dirname(target);
+    this.path = target;
+  }
+
+  public async checkBinary() {
+    const exists = await pathExists(this.path);
+
+    if (exists) {
+      return true;
+    }
+
+    return false;
+  }
+
+  public async download() {
+    if (await this.checkBinary()) {
+      return console.log("Already exists");
+    }
+
+    await ensureDir(path.dirname(this.path), 0o755);
+
+    const file = fs.createWriteStream(this.path);
+
+    console.log(`Downloading lens-k8s-version ${this.version} from ${this.url} to ${this.path}`);
```

Comment:
```suggestion
    console.log(`Downloading lens-k8s-proxy ${this.version} from ${this.url} to ${this.path}`);
```
?
few other places too

Reply:
Good catch! Fixed.